src/OrchardCore.Modules/OrchardCore.Media/Services/MediaOptionsConfiguration.cs

```diff
@@ -61,6 +61,9 @@ public class MediaOptionsConfiguration : IConfigureOptions<MediaOptions>
         // style-src applied to allow browser behaviour of wrapping raw images in a styled img element.
         private const string DefaultContentSecurityPolicy = "default-src 'self'; style-src 'unsafe-inline'";
 
+        private const int DefaultMaxUploadChunkSize = 104_857_600; // 100MB
```

Comment:
```suggestion
        private const int DefaultMaxUploadChunkSize = 104_857_600; // 100MB

```

Reply:
@hishamco I don't see why this new line is required here. Both `DefaultMaxUploadChunkSize` and `DefaultTemporaryFileLifeTime` are for chunked file upload configuration, this is why they are "grouped".

Reply:
To separate constants from fields